source/install/self-managed-changelog.md

```diff
@@ -4,12 +4,133 @@
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.4, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Custom User Groups (Beta; Professional and Enterprise Plans)
+ - Added the ability to mention a group of members at a time in a workspace. Users can create groups, edit group details, join groups, archive groups, and add group members.
+
+#### Cross-team Channel Navigation
+ - You can now find and jump to channels across all your teams by typing any channel name in the **Find Channels** modal.
+
+#### Workspace Optimizations
+ - System Admins can now review their overall health and growth scores and take recommended actions for ensuring their workspace is running smoothly, so users can get the most out of Mattermost.
+
+#### Playbook Updates
+ - Added a new onboarding tour for Playbooks.
+ - Existing playbooks can now be duplicated by copying and modifying your colleague’s processes. Playbooks can also be exported.
+
+#### Boards Updates
+ - Added a new onboarding tour for Boards.
+ - Improved the share boards user interface.
+ - A link to Boards is now included in the channel intro whenever a new channel or direct message is started.
+ - Added in-app links to import Help Docs.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added Persian as an official supported language (Beta).
+ - Added an Enterprise trial activation tour point.
+ - Added a new onboarding experience for first System Admin user.
+ - Added new tutorial tours for Channels, Boards, and Playbooks to help orient first time users.
+ - Removed extra telemetry events that were tracked during page loads.
+ - Added a feature card slide for Playbooks.
+ - Removed ``admin-advisor`` bot's ability to notify admins about missing support email.
+ - Clarified in-product error string "Oops!" as "Unable to continue" for both translators and target audiences in cases where a user doesn't have sufficient permissions to add users or guests.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
+ - Added the ability to send an unsanitized user to the source user on ``user_updated`` event.
+ - In the compact view, the sender's username is now always shown on posts.
+ - The post menu is now only rendered on the root post on hover over.
+ - Updated a library used for storing drafts and other data in browser storage.
+ - Enabled performance telemetry tracking for production deployments not running in developer mode. This telemetry tracking is disabled when telemetry is toggled off.
+ - Inactive server email notifications will now be sent to users occasionally if they haven't interacted with their server for 100 hours or more.
+
+#### Performance
+ - Improved performance of Collapsed Reply Threads when backend is enabled but frontend is disabled.
```

Comment:
Improved database performance when 'ThreadAutoFollow` is enabled, but `CollapsedThreads` is disabled. Learn more about ThreadAutoFollow here > Link to config docs